pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -796,6 +796,8 @@ func (r *Builder) persistentVolumeClaimWithSourceRef(diskAttachment model.XDiskA
 
 	// We add 10% overhead because of the fsOverhead in CDI, around 5% to ext4 and 5% for root partition.
 	// This value is configurable using `FILESYSTEM_OVERHEAD`
+	// Encrypted Ceph RBD causes the pod to see less space, it might be required an overhead in this case.
```

Comment:
```suggestion
	// Encrypted Ceph RBD makes the pod see less space, this possible overhead needs to be taken into account.
```

and wouldn't it make more sense to move this comment to the call to CalculateSpaceWithOverhead?

Reply:
it is.